.npmignore

```diff
@@ -1,2 +1,5 @@
 test/*
-
+.editorconfig
+.eslintrc
```

Comment:
eslintrc and jscs.json must be published, so that `npm run lint` works.

I don't mind excluding the other two, but the benefit is virtually zero imo.